hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/HoodieOptionConfig.scala

```diff
@@ -172,6 +178,15 @@ object HoodieOptionConfig {
     params.get(DataSourceWriteOptions.PRECOMBINE_FIELD_OPT_KEY.key)
   }
 
+  /**
+   * Whether enable the bulk insert for sql insert statement when there is no primaryKey in the table.
+   */
+  def enableBulkInsert(options: Map[String, String]): Boolean = {
```

Comment:
ENABLE_ROW_WRITER_OPT_KEY is a different type of bulk_insert, spark sql support both types?

Reply:
I saw that currently ENABLE_ROW_WRITER_OPT_KEY is only used for bulk insert,  so i reused this config.